src/fastoad/io/configuration/tests/test_configuration.py

```diff
@@ -336,3 +337,35 @@ def test_set_optimization_definition(cleanup):
         conf_dict_opt = conf_dict["optimization"]
         # Should be equal
         assert optimization_conf == conf_dict_opt
+
+
+def test_imports_handling():
+    # Mock the importlib.import_module to return a dummy class
+    class DummyClass:
+        pass
+
+    # Create a dummy module and add it to sys.modules
+    dummy_module_spec = importlib.util.spec_from_loader("my_driver_1", loader=None)
+    dummy_module = importlib.util.module_from_spec(dummy_module_spec)
+    dummy_module.MyDriver1 = DummyClass
+    import sys
+
+    sys.modules["my_driver_1"] = dummy_module
+
+    dummy_module_spec = importlib.util.spec_from_loader("my_driver_2", loader=None)
+    dummy_module = importlib.util.module_from_spec(dummy_module_spec)
+    dummy_module.MyDriver2 = DummyClass
+    sys.modules["my_driver_2"] = dummy_module
```

Comment:
With my suggestion above, you can remove all that part.

Reply:
Corrected in [9d23586](https://github.com/fast-aircraft-design/FAST-OAD/pull/559/commits/9d23586af2929d71c02c34f403f2546d075f6dc7)